Pages/Incoming.cshtml.cs

```diff
@@ -1,218 +1,259 @@
-﻿using System;

```

Comment:
Ugh - not sure what happened here.  Advice to fix?

Reply:
?? I'm not sure what your looking for help with exactly. This is quite a diff.

Reply:
Is it line ending changes? Since you added .gitattributes?

Reply:
Yes, I think so. Will see what I can do.

Reply:
I went through all this recently, it's kinda a pain to deal with.  Even more of a pain if a CVP points out the problem to you 😆 

On macOS/Linux

- add `.gitattributes`
- `rm -rf *`
- `git checkout HEAD .`
- `git add . --renormalize`
- `find . -not -type d -exec file "{}" ";" | grep CRLF`

That last one will tell you if you've got any files that have CRLF.

Reply:
Well that was an adventure, but I think it's fixed now.